File: ClinicSystem/src/com/example/clinicsystem/controllers/C_Employee.java
package com.example.clinicsystem.controllers;

import com.example.clinicsystem.helpers.TextFieldValidator;
import com.example.clinicsystem.models.classes.M_Employee;
import com.example.clinicsystem.models.enums.Gender;

import javax.swing.*;
import java.util.List;

public class C_Employee {


    public List request(String requestType, List<String> list) {
        M_Employee employee = new M_Employee();

        switch (requestType) {
            case "C":
                employee.getAddress().setApartmentNumber(list.get(0));
                employee.getAddress().setCity(list.get(1));
                employee.getAddress().setGovernorate(list.get(2));
                employee.getAddress().setStreetName(list.get(3));
                employee.getAddress().setId(employee.addAddress());

                employee.setBirthDate(list.get(4));
                employee.setFirstName(list.get(5));
                employee.setMiddleName(list.get(6));
                employee.setLastName(list.get(7));
                employee.setPhoneNumber(list.get(8));
                employee.setNationalID(Integer.parseInt(list.get(9)));

                if(list.get(10) == "Male"){
                    employee.setGender(Gender.MALE);
                }else{
                    employee.setGender(Gender.FEMALE);
                }

                int personID = employee.addPerson();

                employee.setEmployeeID(Integer.parseInt(list.get(11)));
                employee.setEmail(list.get(12));

                employee.addEmployee(personID);

                return null;
            case "R":
                if(list == null) {
                    return employee.getAllEmployees("");
                } else {
                    return employee.getAllEmployees(list.get(0));
                }

            case "U":

            case "D":

        }

        return null;
    }

    public boolean isValidEmployeeData(List<JTextField> textFieldList, List<JLabel> labelList) {
        if(TextFieldValidator.validateEmpty(textFieldList, labelList) && TextFieldValidator.validateEmail(textFieldList.get(5), labelList.get(5))) {
            return true;
        }

        return false;
    }
}

File: clinic/C_Employee.php
<?php
include_once "M_Employee.php";
include_once "C_Patient.php";

class C_Employee {
    public function request($requestType, $data) {
        $employee = new M_Employee("", "", "", "", "", null, 1, "", 1, "", 1, "");

        switch ($requestType) {
            case "C":
                $employee->getAddress->setApartmentNumber($data[0]);
                $employee->getAddress->setCity($data[1]);
                $employee->getAddress->setGovernorate($data[2]);
                $employee->getAddress->setStreetName($data[3]);
                $employee->getAddress->setID($employee->addAddress);
                
                $employee->setBirthDate($data[4]);
                $employee->setFirstName($data[5]);
                $employee->setMiddleName($data[6]);
                $employee->setLastName($data[7]);
                $employee->setPhoneNumber($data[8]);
                $employee->setNationalID($data[9]);

                /* if ($data[10] == "Male"){
                    $employee->setGender(Gender.MALE);
                }else{
                    $employee->setGender(Gender.FEMALE);
                } */

                $personID = $employee->addPerson();

                $employee->setEmployeeID($data[11]);
                $employee->setEmail($data[12]);
                
                $employee->addEmployee($personID);

                return null;
            case "R":
				if($data == null) {
                    return $employee->getAllEmployees("");
                } else {
                    return $employee->getAllEmployees($data[0]);
                }
            case "U":

            case "D":
			
			default:
				return null;
        }
    }

    /* public boolean isValidEmployeeData(List<JTextField> textFieldList, List<JLabel> labelList) {
        if(TextFieldValidator.validateEmpty(textFieldList, labelList) && TextFieldValidator.validateEmail(textFieldList.get(5), labelList.get(5))) {
            return true;
        }

        return false;
    } */
}
?>